addons/binding/org.openhab.binding.harmonyhub/META-INF/MANIFEST.MF

```diff
@@ -26,10 +14,24 @@ Import-Package:
  com.google.common.base,
  com.google.common.collect,
  com.google.common.io,
```

Comment:
The `com.google.common` imports can probably be removed now because they were only by the old library.

Reply:
I'll check, you are probably right , i was not sure if GSON required them.